src/engine/target.js

```diff
@@ -20,9 +20,10 @@ class Target extends EventEmitter {
     /**
      * @param {Runtime} runtime Reference to the runtime.
      * @param {?Blocks} blocks Blocks instance for the blocks owned by this target.
+     * @param {?Object.<string, *>} comments Array of comments owned by this target.
```

Comment:
```suggestion
     * @param {?Object.<string, Comment>} comments Array of comments owned by this target.
```

Reply:
There are other places where we used that. Should we fix those too?

Reply:
Yes